pkg/auth/authorizer/abac/BUILD

```diff
@@ -24,9 +24,20 @@ go_library(
     ],
 )
 
+filegroup(
+    name = "example_policy",
+    testonly = True,
+    srcs = [
+        "example_policy_file.jsonl",
+    ],
+)
+
 go_test(
     name = "go_default_test",
     srcs = ["abac_test.go"],
+    data = [
```

Comment:
data is not a managed attribute. src, deps and library are the only managed attributes.